src/graphql/node/crud.rs

```diff
@@ -192,24 +180,32 @@ async fn load(
     super::super::load_edges(&map, after, before, first, last, NodeTotalCount)
 }
 
-/// Returns the customer id of review node.
+/// Returns a customer id and the agent key(agent@hostname) list for the node corresponding to
+/// that customer id.
 ///
 /// # Errors
 ///
 /// Returns an error if the node profile could not be retrieved.
-#[allow(clippy::module_name_repetitions)]
-pub fn get_customer_id_of_node(db: &Store) -> Result<Option<u32>> {
+pub fn get_agent_key_by_customer_id(db: &Store) -> Result<HashMap<u32, Vec<String>>> {
```

Comment:
According to [Rust Naming Conventions](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter), the `get_` prefix is not recommended unless it's necessary. And, not a single key but multiple keys will be returned, so I think `agent_keys_by_customer_id` would be better.

Reply:
done